<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Study Details</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/bootstrap.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Cars</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="speed-and-distance.html">Speed and Distance</a>
</li>
<li>
  <a href="QC.html">QC</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Study Details</h1>

</div>


<p>Data was collected as part of preliminary method development and testing for single-nuclei RNA-sequencing from mouse livers of 2,3,7,8-tetrachlorodibenzo-p-dioxin (TCDD) treated mice. For experimental and model details see the following publications (pubs). A total of 4 samples (2 vehicle, 2 TCDD) were examined by snRNA-seq. Samples were run in two batches (Day 1 - VEH64; Day 2 - VEH62, TCDD51, TCDD59).</p>
<div id="environment-variables-and-packages" class="section level1">
<h1><strong>Environment Variables and Packages</strong></h1>
<pre class="r"><code># Load librariers and set workind directory
library(Seurat)</code></pre>
<pre><code>## Warning: package &#39;Seurat&#39; was built under R version 3.5.3</code></pre>
<p>Analysis was performed using R version 3.5.2 (2018-12-20), Seurat v3.0.2, dplyr v0.8.3, Cowplot v1.0.0, ggplot v3.2.1, and scales v1.0.0. Aligned reads were generated using CellRanger v3.0.2 and aligned to the mm10-3.0.0_premrna mouse build.</p>
</div>
<div id="section-1-data-import-and-preprocessing" class="section level1">
<h1><em><strong>Section 1:</strong></em> <strong>Data import and preprocessing</strong></h1>
<div id="importing-cellranger-data" class="section level2">
<h2><strong>Importing CellRanger data</strong></h2>
<p><span style="background-color: #dbdbdb">Read10X</span> is used to import individual datasets from CellRanger. Datasets are filtered to include only genes detected in at least <strong>3</strong> nuclei, and nuclei which express at least <strong>100</strong> unique genes (features). Nuclei express fewer genes compared to whole cell, therefore these numbers are lower than some single-cell RNA-sequencing datasets. Cells which possess &gt;= <strong>5%</strong> mitochondrial RNA are excluded from future analyses. Nuclei isolation should result in little to no mitochondrial contamination.</p>
<pre class="r"><code>cellmin = 3
featmin = 100
mtmax = 5
#VEH62
VEH62.data &lt;- Read10X(data.dir = &quot;C://Users/Rance/Downloads/NovoG2/VEH62/outs/filtered_feature_bc_matrix/&quot;)
VEH62 &lt;- CreateSeuratObject(counts = VEH62.data, project = &quot;VEH62&quot;, min.cells = cellmin, min.features = featmin)
VEH62[[&quot;percent.mt&quot;]] &lt;- PercentageFeatureSet(VEH62, pattern = &quot;^mt-&quot;)
VEH62$treatment &lt;- &#39;CONTROL&#39;
VEH62 &lt;- subset(VEH62, subset = percent.mt &lt; mtmax)
rm(VEH62.data)
#
#VEH64
VEH64.data &lt;- Read10X(data.dir = &quot;C://Users/Rance/Downloads/NovoG2/VEH64/outs/filtered_feature_bc_matrix/&quot;)
VEH64 &lt;- CreateSeuratObject(counts = VEH64.data, project = &quot;VEH64&quot;, min.cells = cellmin, min.features = featmin) 
VEH64[[&quot;percent.mt&quot;]] &lt;- PercentageFeatureSet(VEH64, pattern = &quot;^mt-&quot;)
VEH64$treatment &lt;- &#39;CONTROL&#39;
VEH64 &lt;- subset(VEH64, subset = percent.mt &lt; mtmax)
rm(VEH64.data)
#
#TCDD51
TCDD51.data &lt;- Read10X(data.dir = &quot;C://Users/Rance/Downloads/NovoG2/TCDD51/outs/filtered_feature_bc_matrix/&quot;)
TCDD51 &lt;- CreateSeuratObject(counts = TCDD51.data, project = &quot;TCDD51&quot;, min.cells = cellmin, min.features = featmin) 
TCDD51[[&quot;percent.mt&quot;]] &lt;- PercentageFeatureSet(TCDD51, pattern = &quot;^mt-&quot;)
TCDD51$treatment &lt;- &#39;TCDD&#39;
TCDD51 &lt;- subset(TCDD51, subset = percent.mt &lt; mtmax)
rm(TCDD51.data)
#
#TCDD59
TCDD59.data &lt;- Read10X(data.dir = &quot;C://Users/Rance/Downloads/NovoG2/TCDD59/outs/filtered_feature_bc_matrix/&quot;)
TCDD59 &lt;- CreateSeuratObject(counts = TCDD59.data, project = &quot;TCDD59&quot;, min.cells = cellmin, min.features = featmin) 
TCDD59[[&quot;percent.mt&quot;]] &lt;- PercentageFeatureSet(TCDD59, pattern = &quot;^mt-&quot;)
TCDD59$treatment &lt;- &#39;TCDD&#39;
TCDD59 &lt;- subset(TCDD59, subset = percent.mt &lt; mtmax)
rm(TCDD59.data)</code></pre>
</div>
<div id="summarizing-datasets" class="section level2">
<h2><strong>Summarizing datasets</strong></h2>
<pre class="r"><code>data.summary &lt;- rbind(dim(VEH62), dim(VEH64), dim(TCDD51), dim(TCDD59))
row.names(data.summary) &lt;- c(&quot;VEH62&quot;, &quot;VEH64&quot;, &quot;TCDD51&quot;, &quot;TCDD59&quot;)
colnames(data.summary) &lt;- c(&quot;Number of Genes&quot;, &quot;Number of Nuclei&quot;)</code></pre>
<p><em><strong>Table 1.</strong></em> Number of genes and nuclei sequenced in liver samples of mice gavaged with sesame oil vehicle or 30 ug/kg TCDD every 4 days for 28 days.</p>
<pre><code>##        Number of Genes Number of Nuclei
## VEH62            17575             3403
## VEH64            17098             6992
## TCDD51           18402             3044
## TCDD59           18607             3272</code></pre>
<p>A total of 16711 (10395 vehicle and 6316 treated) nuclei were transcriptomes were profiled. The average number of genes detected in each sample was 1.7920510^{4} and the median number of unique expressed genes in individual nuclei was 1694 with a median UMI count of with a median UMI count of 3385</p>
</div>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
